data/Incipias/missions.txt

```diff
@@ -0,0 +1,282 @@
+			choice
+				"I am searching for something unique that presents your species best."
+					goto unique
+				"I am gathering proves of friendship from all over the galaxy."
+			`   "That sounds interesting, let me think about it." The Wanderer becomes silent for some time. Then he says, "I got an idea, but I need to ask some people about it first. Please wait here for a second." He leaves the room without waiting for your respond. Forced to wait for him, you ask yourself what this gift of friendship of the wanderers could be.`
+				goto c1
+			label unique
+			`    "Why are you searching for something like this? That is an quite unusual request."`
+			choice
+				"I am a passionate collector."
+				"I want something to remember you."
+			`    "Let me think about this. What would present our species best..." The Wanderer gets silent while thinking. Than he suddenly stands up, "If I get the permission to give you this, you will get a perfect item that presents our species best. Let me go check it." The Wanderer leaves the room.`
+			label c1
+			`    Time is passing and you get a little bit nervous, but after some time the wanderer finally returns. He immedeatly begins to speak. "I just got the permission", he says "You will have to travel to <destination> to get this. You will receive it at the embassy there, I already contacted some people there." You ask him what it his but all he tells you is "It is a surprise, you will see when you get there. Take this." He hands you a small card. "This will tell the people there that you are the right one." He says goodbye to you and you leave the embassy.`
+				accept
+	on complete
+		conversation
+			`    After you arrived at Vara K'chrai, you directly head to the embassy excited what the Wanderers will give you. At the embassy you show the small card the Wanderer gave you and a woman guides you to a separate room, where three Wanderer are standing around a table. On the table is a small plant in a pot. But you have never seen a plant like this before on any Wanderer world. One of the three begins to talk. "This plant on the table there is our gift to you. It is a genmanipulated tree, that will survive under mostly any circumstances, it is something our species is specialized on and it will show our friendship in a very unique way. We hope you like it." Surprised by this unusual gift you thank the wanderers. "It needs nearly no water, so you do not have to care a lot about it, it survives on spaceship so you can take it with you anywhere." You again say thanks to them and then you leave.`
+			`    You now have a prove of the Wanderers friendship. You can show it to the Hai now.`
+
+
+mission "coalition peace prove"
+	landing
+	name "Prove of Peace"
+	source
+		government "Heliarch"
+	to offer
+		"reputation: Heliarch" >= 5
```

Comment:
You don't need to do any of the below, just wanted to point stuff out.

The initial reputation is 1, and incremented by 10 once you complete the First Contact mission.
If that was the intention, you could swap this out for just `has "Coalition: First Contact: done"`. I suppose one could technically get to that reputation by using the Wardragon exploit before completing First Contact, but that'd be very unusual.
Or, if you want to lock it behind something more, you could decide on a number of Coalition jobs needed to successfully get the proof from the Heliarchs.

Reply:
I think I will do this First Contact thing.